boa/src/syntax/ast/node/object/mod.rs

```diff
@@ -8,6 +8,7 @@ use crate::{
 };
 use std::fmt;
 
+use crate::property::{AccessorDescriptor, Attribute, PropertyDescriptor};
```

Comment:
This can be moved up to line 6, becoming just `property::{AccessorDescriptor, Attribute, PropertyDescriptor},`. This way it's all in 1 use statement, like in other files.